container/Dockerfile.trtllm

```diff
@@ -425,39 +430,44 @@ COPY --from=build /usr/local/cuda/lib64/libcudart.so* /usr/local/cuda/lib64/
 COPY --from=build /usr/local/cuda/nvvm /usr/local/cuda/nvvm
 
 # Copy pytorch installation from NGC PyTorch
+# DO NOT INSTALL (networkx packaging setuptools) as those will be overwritten by tensorrt-llm installation later
 ARG TORCH_VER=2.8.0a0+5228986c39.nv25.5
 ARG TORCHVISION_VER=0.22.0a0
-ARG SETUPTOOLS_VER=78.1.1
+# ARG SETUPTOOLS_VER=78.1.1
```

Comment:
nit: Probably better to just remove these `ARG`s and their associated lines to not confuse developers. We can keep the `DO NOT INSTALL` line as the explanation if people encounter build issues related to these packages in the future.